tests/testsuite/config.rs

```diff
@@ -2,12 +2,16 @@ use std::borrow::Borrow;
 use std::collections;
 use std::fs;
 
-use crate::support::{lines_match, paths, project};
+use crate::support::{paths, project};
 use cargo::core::{enable_nightly_features, Shell};
 use cargo::util::config::{self, Config};
 use cargo::util::toml::{self, VecStringOrBool as VSOB};
 use serde::Deserialize;
 
+fn lines_match(a: &str, b: &str) -> bool {
```

Comment:
Can you provide a comment why this is here?  Also, why is it part of this PR, it seems like it should be unrelated?

Reply:
er I mixed up some branches here by accident